playbooks/tasks/portal-role-task-mongo-setup-docker-service.yml

```diff
@@ -6,9 +6,11 @@
   assert:
     that:
       - >-
-        {{ webportal_common_config.mongo_db_mgkey is defined and
-        webportal_common_config.mongo_db_mgkey is not none and
-        webportal_common_config.mongo_db_mgkey != '' }}
+        {{
+          webportal_common_config.mongo_db_mgkey is defined
```

Comment:
nit: If they are all `and` conditionals, i think you can just do a list right?
```yaml
- condition 1
- condition 2
- condition 3
```
I believe ansible interperats lists as `and` conditions by default.

Reply:
Updated.